app/components/Content/ContentHeader.css

```diff
@@ -12,4 +14,8 @@
   display: flex;
   justify-content: flex-end;
   flex-grow: 1;
+
+  @media (--mobile-device) {
+    display: none;
+  }
```

Comment:
What does media here do?? I am intrigued

Reply:
Good question!

This is what we call [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries), and it is essentially how we achieve ["responsive layouts"](https://www.w3schools.com/html/html_responsive.asp). The _condition_ in this case is `--mobile-device`, which is a variable of `max-width: 992px` set in our `variables.css` file. This means that whenever the screen has a width of less than `992px`, the styling set within the brackets will "kick in" (which in this case removes some stuff with the `display: none`). Media queries are very interesting, and a lot more powerful than what we use it for in lego-webapp! It's something we should definitely use more.